dask_geopandas/core.py

```diff
@@ -336,6 +337,33 @@ def hilbert_distance(self, p=15):
 
         return distances
 
+    def morton_distance(self, p=15):
+
+        """
+
+        Parameters
+        ----------
+
+        p : The number of iterations used in constructing the Hilbert curve.
+
+        Returns
+        ----------
+        Distances for each partition
+        """
+
+        # Compute total bounds of all partitions rather than each partition
+        total_bounds = self.total_bounds
```

Comment:
When we update hilbert to allow custom bounds, we should do the same here. Can you make a public note of it somewhere?

Reply:
I've opened a follow up issue in #92